src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -315,4 +315,19 @@
     <value>Allowed value(s): {0}</value>
     <comment>{0} - list of allowed values for an attribute</comment>
   </data>
+  <data name="Error_AtLeastOneSourcePerPackageMode" xml:space="preserve">
+    <value>{0}: Package namespace for '{1}' is not listed on any source. Any package id must be in one or more matching namespace declarations.</value>
```

Comment:
I'm not sure this communicates the problem as clearly. 

I'd lead with something like `Package `{1}` does not match any namespace`.